api/src/routes/pipelines.rs

```diff
@@ -261,17 +259,18 @@ pub async fn read_pipeline(
 
 #[utoipa::path(
     context_path = "/v1",
-    request_body = PostDestinationRequest,
+    request_body = PostPipelineRequest,
     params(
         ("pipeline_id" = i64, Path, description = "Id of the pipeline"),
     ),
     responses(
         (status = 200, description = "Update pipeline with id = pipeline_id"),
-        (status = 404, description = "Pipeline not found"),
-        (status = 500, description = "Internal server error")
-    )
+        (status = 404, description = "Pipeline not found", body = ErrorMessage),
+        (status = 500, description = "Internal server error", body = ErrorMessage)
+    ),
+    tag = "Pipelines"
 )]
-#[post("/pipelines/{pipeline_id:\\d+}")]
```

Comment:
Removed this for consistency with the other endpoints.

Reply:
I remember vaguely this was done to avoid clashing with another endpoint's path. Ok to remove if this doesn't clash was was unnecessary.